source/class/qx/tool/compiler/targets/meta/AbstractJavascriptMeta.js

```diff
@@ -1,3 +1,25 @@
+/* ************************************************************************
+ *
+ *    qooxdoo-compiler - node.js based replacement for the Qooxdoo python
+ *    toolchain
+ *
+ *    https://github.com/qooxdoo/qooxdoo-compiler
```

Comment:
This isn't right any longer, is it? (Might be a global change that should be made in a separate PR.)

Reply:
yes - rather than fix it a bit here and a bit elsewhere, I've done a fix-it-everywhere PR in https://github.com/qooxdoo/qooxdoo/pull/10249